### DISCUSSION TITLE
O-RU support for 7.2 split.

### DISCUSSION THREAD

**Discussion by asheeshtripathi** [srsran/srsRAN_Project#33] [General Help] [answered]
Hi @andrepuschmann, @ismagom 

I am planning to try 7.2 split architecture with CU and DU on different machines and using O-RU. Which O-RU has been tested with the srsRAN Project? And also, please do share if there is any document on how to setup in 7.2 split.

Thanks,
Asheesh

**Comment by rainey19** [comment]
Hi Asheesh,

My current understanding (and what I've gleamed from working with the code) is that the logical splits exist inside the gNB application but they're not currently exposed yet. At this point I think you'd have to implement the socket endpoints and separation yourself for that. There's a sample DU app but I have no idea what's implemented in it, though I might try it with one of our CUs next week.

-Ben

**Comment by andrepuschmann** [comment, marked as answer]
That's right. The open front haul Library is currently under dev and not included in the initial release. We'll release it with one of the upcoming code drops. Also the CU/DU separation is something requested very often - so we're looking into this as well.
We currently use a Benetel 550 RU for the integration.

**Reply by ankuramistry** [reply]
Hi Andre, is there a specific timeline on when the 7.2x split will be available?

**Comment by asheeshtripathi** [comment]
Hi @andrepuschmann @ismagom,
Any update on CU/DU separation and open FH implementation? If I have to start trying CU/DU separation on my own, any suggestions on how to proceed?

**Comment by ismagom** [comment]
Hi @asheeshtripathi,

The OFH implementation will come in the release tomorrow.

The CU/DU split will come after summer. My advise would be to wait until then. The changes are not trivial.

Regards

**Reply by khooi8913** [reply]
Hi @ismagom,

I see that the tutorial to use the O-RU with the OFH implementation is already [available](https://docs.srsran.com/projects/project/en/latest/tutorials/source/oranRU/source/index.html), but it is not included in the 
 current 23.05 release in the [release notes](https://docs.srsran.com/projects/project/en/latest/general/source/2_features_and_roadmap.html).

Is the O-RU support functional at the moment?
Can you kindly comment on that?

Thanks!

-- Xin Zhe

**Reply by ismagom** [reply]
Hi @khooi8913,

The OFH is already available in the repository. The code has not been tagged yet, that's why it doesn't show in the release notes.
